README.md

```diff
@@ -198,6 +198,12 @@ Or run it without zookeeper container:
 
     docker run marathon:{version} --master local --zk zk://localhost:2181/marathon
 
+On Mac you have to use a special address instead of `localhost`:
```

Comment:
Maybe "when using docker for Mac" would be more specific?

Reply:
and we might link https://docs.docker.com/docker-for-mac/networking/#there-is-no-docker0-bridge-on-macos